### ISSUE TITLE
Docs: Rethink display of right panel

### ISSUE THREAD

**Issue by ramirezfranciscof** [aiidateam/aiida-core#4474]
Generated from the discussion in issue #3727

It [was suggested](https://github.com/aiidateam/aiida-core/issues/3727#issuecomment-701652614) that the display of the right panel is not always helpful. Some options (like incorporating it into the left panel) were suggested.

**Comment by chrisjsewell** [comment]
Happy to discuss, but IMO this is a bit of a pointless issue, because it not something that changes to this repository will "fix". Not that I think it needs fixing; as I explained there, this is standard web design.

If you want this changed, you need to open issues on the sphinx theme repository: https://github.com/pandas-dev/pydata-sphinx-theme

**Comment by chrisjsewell** [comment]
I think someone needs to outline exactly what are the criteria for this issue being closed, or it should just be closed

**Comment by sphuber** [comment]
For the record, I tend to agree with @chrisjsewell . This is a question of UX/UI which is notoriously difficult and I don't think we are going to solve it. Even if we might move the right panel elsewhere, which some people may find more intuitive, it is guaranteed that other people will find it less intuitive, taking us back to square one. I would also vote to close this.

**Comment by greschd** [comment]
I think the main issue here is the right panel disappearing when the window is too narrow / on mobile. My preference for an "ideal" UI would be as-is on wide screens, but moving it to the left for mobile. But I think that's tricky to implement, here isn't the right place, and also it's just my preference.
So +1 on the closing.

**Comment by giovannipizzi** [comment]
While I understand that it's going to be tricky to agree on an UX feature, I think that the issue of the panel disappearing on not-so-small screens is an issue for me (we have relatively long pages).

In a sense the 'default' sphinx behaviour is better (IMO), where the left menu expands and you see the submenus in a smaller font.

If it's not possible to configure this behaviour, and decide we accept it, then we should at least open an issue with the original repository and link it here before closing

**Comment by greschd** [comment]
Yeah, I guess we can agree that right panel should go _somewhere_ when the screen becomes narrower.

Whether that means moving it left, collapsing it (with a button to un-collapse) or any other solution is not so important IMO.
